src/Middleware/OutputCaching/samples/OutputCachingSample/Startup.cs

```diff
@@ -0,0 +1,82 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Globalization;
+using Microsoft.AspNetCore.OutputCaching;
+using Microsoft.AspNetCore.OutputCaching.Policies;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddOutputCaching(options =>
+{
+    options.Policies.Add(new EnableCachingPolicy());
+
+    options.Profiles["NoCache"] = new OutputCachePolicyBuilder().NoStore().Build();
+
+    // Tag any request to "/blog**
+    options.Policies.Add(new OutputCachePolicyBuilder().Path("/blog").Tag("blog").Build());
```

Comment:
`Policies` -> `DefaultPolicy`. This should be a `OutputCachePolicy` property instead of a collection.